src/bundle/Resources/config/views.yaml

```diff
@@ -0,0 +1,38 @@
+services:
+    Ibexa\Platform\Search\View\SearchViewProvider:
+        arguments:
+            $matcherFactory: '@platform.user.view.search.matcher_factory.dynamically_configured'
+        tags:
+            - { name: ezpublish.view_provider, type: Ibexa\Platform\Search\View\SearchView, priority: 10 }
+
+    ibexa.platform.view.search.matcher_factory:
+        class: eZ\Bundle\EzPublishCoreBundle\Matcher\ServiceAwareMatcherFactory
+        arguments:
+            $viewMatcherRegistry: '@eZ\Bundle\EzPublishCoreBundle\Matcher\ViewMatcherRegistry'
+            $repository: '@ezpublish.api.repository'
+            $relativeNamespace: Ibexa\Platform\Search\View
+
+    platform.user.view.search.matcher_factory.dynamically_configured:
```

Comment:
This one needs to be aligned as well, I should've marked it, sorry :)

Reply:
:facepalm: :+1: